samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClient.ts

```diff
@@ -102,11 +110,11 @@ const networkRequestViaProxy = <T>(
 
         // establish connection to the proxy
         request.on("connect", (response, socket) => {
-            const statusCode = response?.statusCode || 500;
-            if (statusCode < 200 || statusCode > 299) {
+            const proxyStatusCode = response?.statusCode || ProxyStatus.SERVER_ERROR;
```

Comment:
nit: If the purpose of the same is to keep it simple, I'd remove the proxy stuff from here. If the purpose of the same is for MSAL devs to use it for testing etc, keep it. Not a blocking comment.

Reply:
I'd say the purpose is to be used for testing. I make it clear in the README that this code is copied directly from msal-node's HttpClient. Additionally, if a developer is behind a corporate firewall (at work) they'll need the proxy code.